Rust/src/rcf.rs

```diff
@@ -31,6 +37,20 @@ pub(crate) fn damp(x: usize, y: usize) -> f64 {
 
 const max_number_in_summary : usize = 5;
 
+fn add_to(a: &f64, b : &mut f64){
+    *b += *a;
+}
+
+fn divide(a : &mut f64, b: usize){
+    *a /= b as f64;
+}
+
+fn add_nbr(a:&(f64,usize,f64), b: &mut Vec<(f64,usize,f64)>){
```

Comment:
Is this function used in the same way as the `add_to` and `divide` functions? Why is the first argument mutable in those but the second argument is mutable in this one?

Reply:
The idea of this function is a collector applied to an iterator ... it is convenient if the first is immutable. But a more focused PR can make these in line with the TAR collectors